config/base.js

```diff
@@ -5,6 +5,22 @@ const Job = require('./job');
 const Joi = require('joi');
 const Regex = require('./regex');
 
+const SCHEMA_JOBNAME = Joi.string().regex(Regex.JOB_NAME);
+const SCHEMA_CACHE_VALUE = Joi.string().uri({
+    relativeOnly: true
+});
+const SCHEMA_CACHE_LIST = Joi.array().items(SCHEMA_CACHE_VALUE).min(1);
```

Comment:
I think this should be an object. Order shouldn't matter. 
I'll modify the design doc.